modules/database/src/adapters/DatabaseAdapter.ts

```diff
@@ -63,7 +74,7 @@ export abstract class DatabaseAdapter<T extends SchemaAdapter<any>> {
   ): { model: SchemaAdapter<any>; relations: any };
 
   fixDatabaseSchemaOwnership(schema: ConduitSchema) {
-    const dbSchemas = ['CustomEndpoints'];
+    const dbSchemas = ['CustomEndpoints','_PendingSchemas'];
```

Comment:
@kkopanidis It is required in order to call `createSchemaFromAdapter()` from `Database` (that is to say, without going through gRPC), as otherwise the `ConduitSchema` would get an undefined `ownerModule` and fail the `getSchemaOwnership()` check.

We could also explicitly set the object owner post schema instantiation for database models I guess?